packages/containers/src/SelectObject/SelectObject.component.js

```diff
@@ -16,25 +17,49 @@ function SelectObject(props) {
 			...props.schema,
 		};
 	}
+	const { className = '', ...filter } = props.filter || {};
 	return (
 		<div className={`tc-select-object ${theme.wrapper}`}>
-			<FilterBar id={props.id} dockable={false} navbar={false} {...props.filter} />
+			<FilterBar
+				className={classNames(className, theme.filter)}
+				id={props.id}
+				dockable={false}
+				navbar={false}
+				{...filter}
+			/>
```

Comment:
What about 
```
<FilterBar
	{...filter}
	className={classNames(filter && filter.className, theme.filter)}
	id={props.id}
	dockable={false}
	navbar={false}
/>
```
to avoid the classname + rest filter extraction ?